### DISCUSSION TITLE
Review Transformer Model used in pfcoil module

### DISCUSSION THREAD

**Discussion by ajpearcey** [ukaea/PROCESS#3064] [Ideas]
# Original Issue #101 

> Consider whether transformer flux swing calculation can be improved by using the mutual inductance between the Central Solenoid 
> and the plasma - which is already calculated.
> The description in the Physics paper is as follows, but no justification is given:

![image](https://github.com/ukaea/PROCESS/assets/78224916/0b0d79ea-65d6-47ca-9089-e2e7e4cdf3e3)

# Papercut Review

We should review the models in the pfcoil module are they are quite opaque and current documentation is not complete. Then we can perform a literature review of other possible models

Also we should highlight that PROCESS performs a calculation for both the volt-secs produced by the coil set and compute the volt-sec required for the plasma physics solution. We have a few open discussions on these issue (eg #2884), and it would make sense of consider these issues together.

**Comment by timothy-nunn** [comment]
Worth checking the bit of the code that uses a single value decomposition. When moving to the scipy implementation we got different results. I also showed that the in-house PROCESS algorithm was wrong https://github.com/ukaea/PROCESS/pull/3418#issuecomment-2637277380
